rules/sinks/third_parties/api/java.yaml

```diff
@@ -4,4 +4,5 @@ sinks:
     name: Third Party API
     patterns:   
       - "(?i)((?:http|https):\\/\\/[a-zA-Z0-9_-][^)\\/(#|,!>\\s]{1,50}\\.(?:com|net|org|de|in|uk|us|io|gov|cn|ml|ai|ly|dev|cloud|me|icu|ru|info|top|tk|tr|cn|ga|cf|nl)).*(?<!png|jpeg|jpg|txt|blob|css|html|js|svg)"
+      - "(?i).*((hook|base|authorize|provider|endpoint|installation)(s){0,1}_url|(slack|web)_hook|(rest|api)_endpoint).*"
```

Comment:
We need support for these to be matched with identifiers or members. can you confirm if it is not just literals ?

Reply:
this will only be matched with literals